tests/framework-tests/preferences/js/SelfVoicingPanelTests.js

```diff
@@ -62,6 +63,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 expect: 4,
                 name: "Test the rendering of the speak panel",
                 sequence: [{
+                    event: "{testEnvironment speak}.events.onResourcesFetched",
+                    listeners: "fluid.identity"
+                },  {
```

Comment:
I think we want to report that it actually fired, that is, to use `jqUnit.assert` here instead of `fluid.identity`.